tests/playwright-test/hooks.spec.ts

```diff
@@ -920,3 +920,112 @@ test('afterAll should be run for test.skip', async ({ runInlineTest }) => {
     'afterAll2',
   ]);
 });
+
+test('should respect test.fail.only in beforeEach failure', async ({ runInlineTest }) => {
```

Comment:
Let's remove newly added tests in this file, they are not testing anything substantially new, and just add some maintenance overhead.